### ISSUE TITLE
Question about shuffled data

### ISSUE THREAD

**Issue by Maxwell2017** [shamangary/FSA-Net#48]
Hi shamangary，When I reproduced your code, I found that np.random.shuffle () was found in line 198 of training_and_testing / FSANET_fine_train.py, which would cause the training data for each training session to be inconsistent. I think the training data should be determined in advance and shouldn't shuffle the data in the training code. How do you like it?
https://github.com/shamangary/FSA-Net/blob/3dc37c0bb9dab3875a7386b5a9caa261d35a251d/training_and_testing/FSANET_fine_train.py#L198
